### ISSUE TITLE
Syntax error on Ruby < 2.6

### ISSUE THREAD

**Issue by benjaminwood** [hintmedia/zoom_rb#419]
The code in question `(match[1..]`) uses Ruby 2.6's infinite range syntax.

I believe we intend to support Ruby 2.4, so this should be changed. We should also run our tests against all supported Ruby versions to ensure this cannot happen again.

```
SyntaxError: /app/vendor/bundle/ruby/2.4.0/gems/zoom_rb-1.1.1/lib/zoom/actions.rb:6: syntax error, unexpected ']'00:05
\w+/).map { |match| match[1..].to_sym }
```

**Comment by benjaminwood** [comment]
If it turns out we do not intend to support Ruby < 2.6, let's at least specify a minimum ruby version so folks don't run into this unexpectedly.

**Comment by dil-bparihar** [comment]
can we have a solution for it @kyleboe because i am also trying to use the latest version of it with ruby 2.5 and getting the same error or please tell me the possible solution i could do it from my side. Thanks

**Comment by kyleboe** [comment]
Thanks for your fix for this @dil-bparihar! #447 has been merged and [1.1.7](https://github.com/hintmedia/zoom_rb/releases/tag/v1.1.7) is released!
